src/kernel/arch/x86/paging.zig

```diff
@@ -0,0 +1,122 @@
+const std = @import("std");
+const panic = @import("../../kmain.zig").panic;
+const arch = @import("../../arch.zig").internals;
+const isr = @import("isr.zig");
+const assert = std.debug.assert;
+const MemProfile = @import("../../mem.zig").MemProfile;
+
+const ENTRIES_PER_DIRECTORY = 1024;
+
+const PAGE_SIZE = 0x400000;
```

Comment:
could rename this it PAGE_SIZE_4MB so is more clear

Reply:
Sure, I was going to make that distinction once 4kb paging is working but I might as well do so now.